# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/uucp/uucp-1.06.1-perl-path.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- uucp-1.06.1/contrib/uureroute.perl.orig	2003-05-12 22:40:45.000000000 +0200
+++ uucp-1.06.1/contrib/uureroute.perl	2003-05-12 22:41:27.000000000 +0200
@@ -1,5 +1,5 @@
-#!/usr/local/bin/perl
-eval ' exec /usr/local/bin/perl $0 "$@" '
+#!/usr/bin/perl
+eval ' exec /usr/bin/perl $0 "$@" '
 	if $running_under_some_shell;
 
 # From a script by <Bill.Campbell@celestial.com>
@@ -34,7 +34,7 @@
 #
 ";
 
-$UUSTAT = "/usr/local/bin/uustat";
+$UUSTAT = "/usr/bin/uustat";
 $SHELL = "/bin/sh";
 $SMAIL = "/bin/smail";
 
